aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -228,8 +228,8 @@ Tensor empty_like(
       // Copy the tensors with channels to avoid accidental overrides
       return at::_empty_per_channel_affine_quantized(
           self.sizes(),
-          self.q_per_channel_scales().clone(),
-          self.q_per_channel_zero_points().clone(),
+          self.q_per_channel_scales().clone(at::MemoryFormat::Contiguous),
```

Comment:
Please make preserve _empty_per_channel_affine_quantized already cast inputs to contiguous. But it might later support other memory formats.

Reply:
changed to Preserve